assets/questions/sortItinerary.js

```diff
@@ -0,0 +1,159 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Sort Scrambled Itinerary.
+ */
+
+globalData.questions['sortItinerary'] = {  // eslint-disable-line dot-notation
+  title: 'Sort Scrambled Itinerary',
+  starterCode: {
+    python:
+`def sortItinerary(tickets):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def connectTicketsInGivenOrder(cls, s):
+        return "-".join(s.split(","))
+    
+    @classmethod
+    def createLongLinearItinerary(cls, atom, input_size):
+        if input_size <= 1: return "0-1"
+        
+        locations = [str(i) for i in xrange(input_size)]
+        tickets = []
+        for i in xrange(0, input_size-1):
+          ticket = "-".join(locations[i:i+2])
+          tickets.append(ticket)
+        
+        linear_combined_tickets = ",".join(tickets)
+        return linear_combined_tickets
+`
+  },
+  tasks: [{
+    instructions: [
+      [
+        'In this question, you will implement the sortItinerary function ',
+        'that sort scrembled airline tickets. ',
+        'It takes a string that represents airline tickets as input. ',
+        'The string can be split by comma into substrings that looks like ',
+        '"XXX-YYY". This substring represents a ticket and means the plane ',
+        'departs from XXX and arrives at YYY, noticing that departure and ',
+        'destinations are not necessarily of length 3. ',
+        'As an example, "LAX-JFK,JFK-ATL" means the itinerary has two flights',
+        ': one from LAX to JFK and the other from JFK to ATL. '
+      ].join(''),
+      [
+        'The function needs to return a string of the sorted itinerary. ',
+        'Locations need to be connected by dash. For the above example, ',
+        'the return string should be "LAX-JFK-ATL". '
+      ].join(''),
+      [
+        'All tickets must be used to form an itinerary. ',
+        'For now, assume there is no loops in the itinerary and is guaranteed ',
+        'to have one and only one itinerary given the list of tickets. '
+      ].join('')
+    ],
+    prerequisiteSkills: ['Graph', 'String Manipulation'],
+    acquiredSkills: ['Graph Traversal', 'Topological Sorting'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'sortItinerary',
+    correctnessTests: [{
+      input: 'LAX-JFK',
+      allowedOutputs: ['LAX-JFK']
+    }, {
+      input: 'JFK-ATL,LAX-JFK',
+      allowedOutputs: ['LAX-JFK-ATL']
+    }, {
+      input: 'MUC-LHR,JFK-MUC,SFO-SJC,LHR-SFO',
+      allowedOutputs: ['JFK-MUC-LHR-SFO-SJC']
+    }],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.connectTicketsInGivenOrder',
+      messages: [
+        "Tickets order are scrambled. Just connect them in given order ",
```

Comment:
This isn't consistent with the tone of existing questions. Please check the questions that are already in the codebase and update them to make sure that your tone is consistent with what we're trying to go for.